src/web/nextui/src/app/eval/ResultsView.tsx

```diff
@@ -550,7 +550,8 @@ export default function ResultsView({
                 Table Settings
               </Button>
             </Tooltip>
-            {config?.metadata?.redteam && (
+            {/* TODO: Remove config.metadata.redteam check in favor of config.redteam */}
```

Comment:
nit - I always put a name to TODOs (indicates author, not necessarily the one to fix it) and for deprecations I usually put a date 
```suggestion
            {/* TODO(Michael): Remove config.metadata.redteam check (2024-08-18) */}
```